sycl/test/regression/bit_cast_lin.cpp

```diff
@@ -0,0 +1,4 @@
+// RUN: %clangxx -fsycl -fsycl-host-compiler=g++ -fsycl-host-compiler-options='-std=c++17' %s -o %t.out
```

Comment:
Why do we need to change host compiler?

Reply:
The problem was reproducible only if host compiler was g++